Berksfile

```diff
@@ -1,3 +1,5 @@
 source 'https://supermarket.chef.io'
 
+cookbook 'fedora-test', path: 'test/cookbooks/fedora-test'
```

Comment:
This cookbook shouldn't be needed since you can do this with attributes. Can you please do that in the kitchen.yml instead?

Reply:
yep